.dockerignore

```diff
@@ -16,4 +16,6 @@ crates/stark_hash_python/target/
 .envrc
 .env
 # pyenv
-.python-version
\ No newline at end of file
+.python-version
+nodes/Dockerfile.pathfinder
+nodes/docker-compose.yaml
```

Comment:
Can't we just ignore the complete `nodes` directory here?

Reply:
Yeah, that's why I was suggesting using `PATHFINDER_FORCE_VERSION` instead of this hack. That way we don't need the `.git` directory at all.